src/service/tar_git

```diff
@@ -787,7 +787,7 @@ clone () {
 
   echo "Handling $CLONE_NAME"
   URL=$MYURL
-  cache_prefetch $CLONE_NAME $URL
+  cache_prefetch $URL
   MYURL=$URL
```

Comment:
I was afraid for CLONE_NAME we'd lose the logic of parse_url, but basename nicely does the same thing \o/